paper/header.tex

```diff
@@ -9,9 +9,15 @@
 \author[1, 3]{\\ Davi M. Valladão}
 \author[4]{Oscar Dowson}
 \affil[1]{Laboratory of Applied Mathematical Programming and Statistics (LAMPS)}
+% <review @frapac>
+% Eletrical -> Electrical
+% </review>
```

Comment:
Also 
we should update `\author[1, 2, 5]{Joaquim D. Garcia}`
and `\affil[5]{PSR-Inc}`